tests/phpunit/tests/theme/base.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+abstract class WP_Theme_UnitTestCase extends WP_UnitTestCase {
```

Comment:
Why add this base test case? Test classes in this group are repeating the same code. This base abstract test case encapsulates the repeating code, removing it from the test class itself. It's the starting point for improving the other tests in this same group.

Note: A follow-up PR will address the other tests in this group.